public/app/core/utils/explore.ts

```diff
@@ -215,8 +215,9 @@ export function parseUrlState(initial: string | undefined): ExploreUrlState {
     return errorResult;
   }
 
-  // Explore versions before refresh picker
-  if (parsed.length === 5) {
+  // SegmentsStart used to be on RefreshInterval's position. Small hack
```

Comment:
@davkal This way or revert. Whats your opinion?

Reply:
IMO I'd remove this from the URL state until we have redone the URL state handling. We have one format of legacy URLs now, adding this would make it 2 formats, plus the future format.

I'd also argue that refresh state is less important than range or queries, because it's used less and when it's used, it's an active choice during debugging, so unlikely needed when passing URLs around. The only downside I can think of is that you refresh the browser tab by mistake and then the state is gone, but we can fix that in a more comprehensive URL state rewrite.